### PR TITLE
Install kustomize without snap

### PR DESCRIPTION
**Pull request by paolomainardi**
# Description

New tool, Bug fixing, or Improvement?  Improvement

Install kustomize without Snap, as documented here: https://github.com/actions/virtual-environments/issues/1514

## Check list
- [x]  https://github.com/actions/virtual-environments/issues/1514
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/kubernetes-tools.sh [hunk 1]
```diff
@@ -24,7 +24,9 @@ curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-
 sudo install minikube-linux-amd64 /usr/local/bin/minikube
 
 # Install kustomize
-sudo snap install kustomize
+curl -s "https://raw.githubusercontent.com/\
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/kubernetes-tools.sh:27]: Could you please move url to the separate variable for readability?
```
download_url="https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
curl -s $download_url | bash
mv kustomize /usr/local/bin
```

Comment (paolomainardi) [reply images/linux/scripts/installers/kubernetes-tools.sh]: done

**Review by miketimofeev — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Comment by paolomainardi** [comment]
Thank you all for the the review

**Pull request merged**